Makefile

```
@@ -33,54 +37,29 @@ sls-only-deploy: # Deploys all lambdas - mandatory: PROFILE, VERSION=[commit has
 
 undeploy: # Undeploys whole project - mandatory: PROFILE
 	make serverless-remove VERSION="any"
```

Add this line here please
```suggestion
	make serverless-remove VERSION="any"
	make terraform-destroy-auto-approve STACKS=lambda-security-group
```